tests/run_bf.sh

```diff
@@ -15,4 +15,4 @@ else
 fi
 export PYTHONPATH
 
-exec python3 "${bf_dir}"/bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json -n prplmesh_docker -x test_flows
+exec bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json -n prplmesh_docker -x test_flows
```

Comment:
When Oleksii originally integrated boardfarm, I asked *not* do to this exactly because it requires installing boardfarm, which I wanted to avoid :-)

But I guess it's OK now that we have it in docker.

Reply:
This change is actually bogus; I added a script run_bf_compose.sh that still uses the boardfarm in it's local copy, as above